exercises/roman-numerals/canonical-data.json

```diff
@@ -93,7 +98,7 @@
     },
     {
       "uuid": "d5b283d4-455d-4e68-aacf-add6c4b51915",
-      "description": "50 is a single L",
+      "description": "59 is a single LIX",
```

Comment:
Do we need the `is a single` bit in the description? It looks like it was removed from other test cases.

Reply:
Whoops, I missed this in the update.
Well caught Erik